adagrams/game.py

```diff
@@ -1,11 +1,113 @@
+import random
+
+DISTRIBUTION_OF_LETTERS = {
+    "A": 9,
+    "B": 2,
+    "C": 2,
+    "D": 4,
+    "E": 12,
+    "F": 2,
+    "G": 3,
+    "H": 2,
+    "I": 9,
+    "J": 1,
+    "K": 1,
+    "L": 4,
+    "M": 2,
+    "N": 6,
+    "O": 8,
+    "P": 2,
+    "Q": 1,
+    "R": 6,
+    "S": 4,
+    "T": 6,
+    "U": 4,
+    "V": 2,
+    "W": 2,
+    "X": 1,
+    "Y": 2,
+    "Z": 1,
+}
+
+SCORE_CHART = {
+    "A": 1,
+    "B": 3,
+    "C": 3,
+    "D": 2,
+    "E": 1,
+    "F": 4,
+    "G": 2,
+    "H": 4,
+    "I": 1,
+    "J": 8,
+    "K": 5,
+    "L": 1,
+    "M": 3,
+    "N": 1,
+    "O": 1,
+    "P": 3,
+    "Q": 10,
+    "R": 1,
+    "S": 1,
+    "T": 1,
+    "U": 1,
+    "V": 4,
+    "W": 4,
+    "X": 8,
+    "Y": 4,
+    "Z": 10,
+}
+
+
 def draw_letters():
-    pass
+    letters_drawn = []
+    available_letters = []
+    for letter, limit in DISTRIBUTION_OF_LETTERS.items():
+        available_letters.append(letter)
+    while len(letters_drawn) <= 9:
+        letter_choice = random.choice(available_letters)
+        letter_count = letters_drawn.count(letter_choice)
+        if letter_count < DISTRIBUTION_OF_LETTERS[letter_choice]:
+            letters_drawn.append(letter_choice)
+    return letters_drawn
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+    for letter in word:
+        letter = letter.upper()
```

Comment:
[nit] You don't need this because `word.upper()` just guaranteed every `letter` will already be uppercase.

```suggestion
```